src/kohlrahbi/conditions/command.py

```diff
@@ -5,7 +5,7 @@
 It takes several options as command-line arguments and performs the necessary operations based on those options.
 
 Options:
-- `--edi-energy-mirror-path`: Specifies the path to the edi_energy_mirror repository.
+- `--input-path`: Specifies the path to the folder with the docx AHBs.
```

Comment:
in the README you changed it to `edi-energy-mirror-path` 
I thought we use the `edi-energy-mirror-path` as a flag

Reply:
59127c985acd8f23c52090a0a87197a439ce9636